import/popups/PopupOpener.qml

```diff
@@ -11,11 +11,11 @@
 
 import QtQuick 2.2
 import Sailfish.WebView.Popups 1.0 as Popups
+import "GeolocationHosts.js" as Geolocation
 
 Timer {
     id: root
 
-    property var geolocationUrls: {{}}
```

Comment:
Regarding the issue this caused the undefined geolocationUrls errors. As I noticed that geolocation requests are not shared across tabs fixed that at the same go.

Should have been like:
```
property var geolocationUrls: ({})
```